src/vs/workbench/contrib/welcome/gettingStarted/browser/gettingStarted.contribution.ts

```diff
@@ -235,7 +235,7 @@ ExtensionsRegistry.registerExtensionPoint({
 							},
 							description: {
 								type: 'string',
-								description: localize('walkthroughs.tasks.description', "Description of task. Use markdown-style links fo commands or external links: [Title](command:myext.command), [Title](command:toSide:myext.command), or [Title](https://aka.ms)")
+								description: localize('walkthroughs.tasks.description', "Description of task. Supports ``preformatted``, __italic__, and **bold** text. Use markdown-style links for commands for external links: [Title](command:myext.command), [Title](command:toSide:myext.command), or [Title](https://aka.ms). Links on their own line will be rendered as buttons.")
```

Comment:
Would pre-formatted text use a single or double backtick? To match markdown, I think it should probably be a single one

Reply:
We don't match markdown already, and this is more consistant with the existing... also it seems like markdown is happy with two backtick, even though one also works:

Markdown: *one asterisk*, **two asterisk**, _one underscore_, __two underscore__, `one backtick`, ``two backtick``

Us: \*one asterisk\*, *two asterisk*, \_one underscore\_, __two underscore__, \`one backtick\`, `two backtick`

Reply:
Ok that's probably ok then and reduces the chance of accidentally creating a code block. I was confused because markdown does support `**` and `__` (but, as you noted, with different meanings than our formatter)

Reply:
Yeah, I prefer our approach TBH, I always end up making weird italic markdown sections when trying to type out mathematic expressions in md-enabled inputs, for instance.